README.md

```diff
@@ -89,11 +89,17 @@ gulp.task('sass', function () {
 
 ```
 
-To import images, configure your application to reference or copy the icons assets.
+### Import images and icons
 
-Below is a code sample you can add to your [express.js](https://expressjs.com/) configuration:
+In order to import GOV.UK Frontend images and icons to your project, you should configure your application to reference or copy the relevant GOV.UK Frontend assets.
+
+You can do this with one of the following:
+
+1. Set `$global-images` variable in `/node_modules/frontend/global/settings` to point to your images folder.
```

Comment:
1. This is pointing to the settings directory but we probably want to be referencing the `_paths.scss` file within that directory.
2. This makes it sound like they should _change the file in mode_modules_ but they can't and shouldn't do that – we want them to set the `$global-images` in their own SCSS file, _before_ they import Frontend.
3. That variable should probably be prefixed… 🤔

Reply:
I've prefixed $global-images and sneaked it into this PR 😁 Shout if you think it needs its own PR.